Parsers/ForgeRockParser.txt

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Please add parser notes similar to other parsers under Parser folder. description, updated by , instruction to create function etc.